GPU/Common/TextureCacheCommon.cpp

```diff
@@ -2095,8 +2108,18 @@ bool TextureCacheCommon::PrepareBuildTexture(BuildTexturePlan &plan, TexCacheEnt
 		}
 	}
 
-	if (plan.badMipSizes) {
+	bool isVideo = IsVideo(entry->addr);
+
+	// TODO: Support reading actual mip levels for upscaled images, instead of just generating them.
+	// Probably can just remove this check?
+	if (plan.scaleFactor > 1) {
 		plan.maxLevel = 0;
+
+		bool enableVideoUpscaling = false;
```

Comment:
I could imagine we might want video upscaling with hardware shader scaling.  Maybe.

-[Unknown]

Reply:
Videos are not really the kind of content that scales well with this approach, at least. But yeah, could experiment with it, but for now I'm keeping it disabled.